input/data/2013/2013A%26A...551A..94H/tev-000060.yaml

```diff
@@ -0,0 +1,25 @@
+source_id: 60
+reference_id: 2013A&A...551A..94H
+telescope: hess
+
+data:
+  livetime: 6h
+  n_on: 112
+  n_off: 365
+  alpha: 0.077
+
+# No position given in the paper
+
+morph:
+  type: point
+
+#############################################################
+# HESS January - March 2011
+
+spec:
+  model:
+    type: pl
+    parameters:
+      norm: {val: 1.95, err: 0.32, err_sys: 0.39, scale: e-12, unit: cm-2 s-1 TeV-1}
```

Comment:
There's a formatting error on this line. It should be `scale: 1e-12`:
https://travis-ci.org/gammapy/gamma-cat/builds/298950008#L1324
Please run `make.py all` locally to make sure all is well after doing data entry.